File: testapp/src/main/java/com/tokopedia/tkpd/network/LogoutPojo.java
package com.tokopedia.tkpd.network;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by hendry on 30/01/19.
 */
public class LogoutPojo {
    @SerializedName("data")
    public Data data = null;
    @SerializedName("message_error")
    public List<String> message_error = new ArrayList<>();
    @SerializedName("message_status")
    public List<String> message_status = new ArrayList<>();

    public static class Data {
        @SerializedName("is_logout")
        boolean is_logout =false;
    }
}

File: features/user/logout/src/main/java/com/tokopedia/logout/domain/pojo/LogoutPojo.kt
package com.tokopedia.logout.domain.pojo

import com.google.gson.annotations.SerializedName

/**
 * @author by nisie on 5/30/18.
 */
data class LogoutPojo(
        @SerializedName("data")
        val data: Data? = null,
        @SerializedName("message_error")
        val message_error: List<String>? = ArrayList(),
        @SerializedName("message_status")
        val message_status: List<String>? = ArrayList()
)

data class Data(
        @SerializedName("is_logout")
        val is_logout: Boolean = false
)